external/fv3fit/fv3fit/pytorch/graph/train.py

```diff
@@ -117,7 +114,7 @@ def train_graph_model(
     predictor = PytorchAutoregressor(
         state_variables=hyperparameters.state_variables,
         model=train_model,
-        scalers=cast(Mapping[Hashable, StandardScaler], scalers),
+        scalers=scalers,
```

Comment:
It was necessary to switch to str for variable names so that I can store separate "a_" and "b_" scalers for the input and outpoint domain in one scalers mapping. Technically this isn't needed by PytorchAutoregressor, but it shares dumping/loading code with the CycleGAN reloadable to reduce code duplication. There doesn't appear to be any loss of interoperability with xarray by doing so, all predict methods still use xr.Dataset.